multinode-demo/drone.sh

```diff
@@ -7,18 +7,40 @@ here=$(dirname "$0")
 # shellcheck source=multinode-demo/common.sh
 source "$here"/common.sh
 
-usage() {
-  if [[ -n $1 ]]; then
-    echo "$*"
-    echo
+usage () {
+  exitcode=0
+  if [[ -n "$1" ]]; then
+    exitcode=1
+    echo "Error: $*"
   fi
-  echo "usage: $0]"
-  echo
-  echo " Run an airdrop drone"
-  echo
-  exit 1
+  cat <<EOF
+usage: $0 [-c lamport_cap]
+
+Run an airdrop drone
+
+ -n lamport_cap    - Airdrop cap in lamports
+
+EOF
+  exit $exitcode
 }
 
+lamport_cap=1000000000
+
+while getopts "h?c:" opt; do
```

Comment:
How about we just pass through all additional arguments directly to the real drone and nuke all this getopt wrapper stuff

Reply:
@mvines How's this?

Reply:
It looks like we can remove that `usage` function entirely